pyproject.toml

```diff
@@ -51,7 +51,7 @@ warn_no_return = "False"
 module = [
     "pytorch_lightning.callbacks.progress.rich_progress",
     "pytorch_lightning.core.datamodule",
-    "pytorch_lightning.demos.boring_classes",
+    "pytorch_lightning.core.module",
```

Comment:
Why are you adding this file?

Reply:
Sorry, that's my merge gonna wrong. Fixed now